lib/Widget/dashboard/dashboard_Table.dart

```diff
@@ -11,12 +13,12 @@ class DashboardTable extends StatefulWidget {
 
 class _DashboardTableState extends State<DashboardTable> {
   final CustomMaterial _customMaterial = locator<CustomMaterial>();
-  List<Map<String, dynamic>> test = [
+  List<Map<String, dynamic>> dashboardtable = [
     {
       "Date": "10/03/64",
       "Time": "02.00 น.",
       "Admission": 1,
-      "Operation": "",
+      "Operation": "0",
       "RespirationsRate": 14,
       "BloodPressure": "125/80",
       "O2Sat": 100,
```

Comment:
Is this map comefrom db  ?

Reply:
It will come from the number of days after the surgery since post-op@hos day0
I think we will calculate by latestStateChanged like the app side?